hw_diag/utilities/quectel.py

```diff
@@ -18,50 +19,58 @@
 logging = get_logger(__name__)
 
 # quectel unique properties
-EG25G_UNIQUE_PROPERTIES = {'Revision': ['EG25GGBR07A08M2G', 'EG25GGBR07A07M2G']}
+EG25G_UNIQUE_PROPERTIES = {"Revision": ["EG25GGBR07A08M2G", "EG25GGBR07A07M2G"]}
 MODEM_RESET_WAIT_TIME = 120
 INTERNET_MAX_WAIT_TIME = 600
 
 # latest firmware that is suppoed to fix AT&T 3G shutdown.
 EG25G_OLD_KNOWN_FW = {
-    'EG25GGBR07A08M2G': 'EG25GGBR07A08M2G_01.001.01.001',
-    'EG25GGBR07A07M2G': 'EG25GGBR07A07M2G_01.001.01.001'
+    "EG25GGBR07A08M2G": "EG25GGBR07A08M2G_01.001.01.001",
+    "EG25GGBR07A07M2G": "EG25GGBR07A07M2G_01.001.01.001",
 }
 
 EG25G_DESIRED_FW = {
-    'EG25GGBR07A08M2G': 'EG25GGBR07A08M2G_30.006.30.006',
-    'EG25GGBR07A07M2G': 'EG25GGBR07A07M2G_30.003.30.003'
+    "EG25GGBR07A08M2G": "EG25GGBR07A08M2G_30.006.30.006",
+    "EG25GGBR07A07M2G": "EG25GGBR07A07M2G_30.003.30.003",
 }
 
 FW_FILE_HASH = {
-    'EG25GGBR07A07M2G_01.001.01.001.tgz': 'db59e2afd2a9497f3750366da6d42a40',
-    'EG25GGBR07A07M2G_30.003.30.003.tgz': '914dd0a491ac212f417d1d286a8146ad',
-    'EG25GGBR07A08M2G_01.001.01.001.tgz': '6c22bb953d7b57371e8326b341938d10',
-    'EG25GGBR07A08M2G_30.006.30.006.tgz': '08fc2173a2582a6cef2d62ca8a169d1a'
+    "EG25GGBR07A07M2G_01.001.01.001.tgz":
+        "180bf41a3f08e5eb631d1ee98d1ff18dca14e6ef32ef9d41f704cc261ad163d0",
+    "EG25GGBR07A07M2G_30.003.30.003.tgz":
+        "7220c074d2abdcfa1d5e5740f9c8ad5561f09f294ce4e36bb6e05c9e3c0762dc",
+    "EG25GGBR07A08M2G_01.001.01.001.tgz":
+        "00c21f7843a12923265628a619fbea77876e393d51e94cf8f47e68aebf32a038",
+    "EG25GGBR07A08M2G_30.006.30.006.tgz":
+        "bde8796ae6307ecd31dce477d1a9281f67339ed0fbe4eee2d1ab108a061243b7",
 }
 
-GCP_FW_BASEPATH = ('https://storage.googleapis.com/helium-assets.nebra.com/'
-                   'modem-firmware/eg25g_firmware/')
-FW_STATE_FILE = '/var/data/quectel_state'
-FW_STORE_PATH = '/var/data/modem_firmware'
+GCP_FW_BASEPATH = (
+    "https://storage.googleapis.com/helium-assets.nebra.com/modem-firmware/eg25g_firmware/"
```

Comment:
@shawaj @vpetersson this ultimately is used for the path to a `.tgz` with the firmware. There won't be nearly as much volume as the snapshots. I don't think it's necessary to switch this to nebracdn and may actually cause more confusion if they ask if this is "web traffic".

Reply:
We will continue to use `helium-assets.nebra.com`